Editing mulltiple accounts page with environment info (#1610)
advancedtelematic/aktualizr: docs/ota-client-guide/modules/ROOT/pages/intro-prep.adoc

Here are our recommended steps for integration:

* xref:add-environments.adoc[*Set up different environments*]
+

> Avoid mixing up test software and production software by uploading the software under different environments.

Reviewer: ```suggestion
To avoid mixing up test software and production software, upload the software to different environments.
```

* xref:add-environments.adoc[*Set up different environments*]
+

> To avoid mixing up test software and production software upload the software to different environments.

Reviewer: ```suggestion
To avoid mixing up test software and production software, upload the software to different environments.
```

Reviewer: The comma is still missing :)

Author: I think it wasn't saved in my commit. I'll try it again.
